src/components/Disclosure/Disclosure.tsx

```diff
@@ -11,7 +11,7 @@ import {b} from './cn';
 import './Disclosure.scss';
 
 export type DisclosureSize = 'm' | 'l' | 'xl';
-export type DisclosureArrowPosition = 'left' | 'right' | 'start' | 'end';
```

Comment:
It would break typings for those who passes this type to the props. Let's keep them here. I will add warning later, for all such types